pkg/scheduler/scheduler.go

```diff
@@ -126,6 +134,7 @@ func New(queues *queue.Manager, cache *cache.Cache, cl client.Client, recorder r
 		preemptor:               preemption.New(cl, wo, recorder, options.fairSharing, realClock),
 		admissionRoutineWrapper: routine.DefaultWrapper,
 		workloadOrdering:        wo,
+		clock:                   realClock,
```

Comment:
You steal using realClock. Use clock from options.

Reply:
I didn't understand.... Please elaborate